src/videojs-fairplay.js

```diff
@@ -128,11 +129,42 @@ class Html5Fairplay {
   onCertificateLoad(event, { callback }) {
     this.log('onCertificateLoad()');
 
-    certificate = new Uint8Array(event.target.response);
+    const {
+      response,
+      status,
+    } = event.target;
+
+    if (status !== 200) {
+      this.onRequestError(event.target, ERROR_TYPE.FETCH_CERTIFICATE);
+
+      return;
+    }
+
+    certificate = new Uint8Array(response);
 
     callback();
   }
 
+  onRequestError(request, errorType = ERROR_TYPE.UNKNOWN) {
+    this.log('onRequestError()');
+
+    const errorMessage = `${errorType} - DRM: com.apple.fps.1_0 update, 
```

Comment:
Does this error message get properly formatted or does it render weird due to the ```?